app/pages/homepage/ui.test.js

```diff
@@ -226,6 +252,19 @@ test('should render the compare promo', async (t) => {
     .expect(await extractInnerText(promo.find('p'))).eql(content.comparePromoDescription);
 });
 
+test('should navigate to the compare page when the compare promo is clicked', async (t) => {
+  await pageSetup({ t, cookiePayload: { id: '88421113', name: 'Cool Dude', organisation: 'view' } });
```

Comment:
can we remove the cookiePayload. Not really needed for this. Looks bit misleading

Reply:
oh yeah I removed it from the others